simphony_metadata/scripts/generate.py

```diff
@@ -29,8 +27,8 @@
 # FIXME: These are excluded because they are not defined CUBA keys
 EXCLUDE_SUPPORTED_PARAMETERS = ('definition', 'models', 'variables', 'data',)
 
-# validation.py for validation codes
-VALIDATION_PY_PATH = os.path.splitext(validation.__file__)[0]+'.py'
+# validation.py for validation codes.
+VALIDATION_PY_PATH = 'simphony_metadata/scripts/validation.py'
```

Comment:
Would this prevent the click command from being invoked in other directories?  Alternatively you can use `inspect` to get the path to this generate.py file and then compose the path to the validation.py